Sources/TuistAcceptanceTesting/TuistAcceptanceTest.swift

```diff
@@ -1,96 +1,164 @@
-import ArgumentParser
-import FileSystem
-import Foundation
 import Path
 import Testing
 import TuistSupport
-import TuistSupportTesting
-@testable import TuistKit
+import XcodeProj
 
-public struct TuistAcceptanceTestFixtureTrait: TestTrait, SuiteTrait, TestScoping {
-    let fixturePath: AbsolutePath
+enum TuistAcceptanceTest {
```

Comment:
```suggestion
public enum TuistAcceptanceTest {
```

Otherwise, its methods being public has no effect. Alternatively, we can make these an extension of `Suite`.